notebooks/111-detection-quantization/111-detection-quantization.ipynb

```diff
@@ -0,0 +1,773 @@
+    "    int8_net = ie.read_network(compressed_model_path)\n",
+    "    int8_exec_net = ie.load_network(network=int8_net, device_name=\"CPU\")\n",
+    "    input_layer = next(iter(int8_net.input_info))\n",
+    "    output_layer = next(iter(int8_net.outputs))\n",
+    "    int8_res = int8_exec_net.infer({input_layer: input_image})[output_layer]\n",
+    "    int8_metric = MAPMetric(map_value=map_value)\n",
+    "    int8_metric.update(output=[int8_res], target=[target_annotation])\n",
+    "    \n",
+    "    for item in int8_res[0, 0, ::]:\n",
+    "        _, _, conf, xmin, xmax, ymin, ymax = item\n",
+    "        if conf > confidence_threshold:\n",
+    "            total_image = draw_boxes_on_image(\n",
+    "                [xmin, xmax, ymin, ymax], total_image, \"yellow\"\n",
+    "            )\n",
+    "            int8_image = draw_boxes_on_image(\n",
+    "                [xmin, xmax, ymin, ymax], orig_image, \"yellow\"\n",
+    "            )\n",
+    "            \n",
+    "    axs[i, 2].imshow(int8_image)\n",
+    "\n",
+    "    # Annotation\n",
+    "    for annotation in target_annotation:\n",
+    "        total_image = draw_boxes_on_image(annotation[\"bbox\"], total_image, \"green\")\n",
+    "\n",
+    "    axs[i, 0].imshow(image)\n",
+    "    axs[i, 0].set_title(Path(metadata[\"filename\"]).stem)\n",
+    "    axs[i, 1].set_title(f\"FP32 mAP: {fp_metric.avg_value[map_value]:.3f}\")\n",
+    "    axs[i, 2].set_title(f\"INT8 mAP: {int8_metric.avg_value[map_value]:.3f}\")\n",
+    "    fig.suptitle(\n",
+    "        \"Annotated (green) and detected boxes on FP (red) and INT8 (yellow) model\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c18abd6-393e-4376-a3fc-d389e5b73ff3",
+   "metadata": {},
+   "source": [
+    "## Compare Size of Original and Quantized Models"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d599fa7-bf93-4476-a183-7cad01d2e4ed",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "original_model_size = Path(ir_path).with_suffix(\".bin\").stat().st_size / 1024\n",
+    "quantized_model_size = (\n",
+    "    Path(compressed_model_path).with_suffix(\".bin\").stat().st_size / 1024\n",
+    ")\n",
+    "\n",
+    "print(f\"FP32 model size: {original_model_size:.2f} KB\")\n",
+    "print(f\"INT8 model size: {quantized_model_size:.2f} KB\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "322e899a-5bcf-4759-964e-9c9721d39928",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Compare Performance of the Original and Quantized Models\n",
+    "\n",
+    "To measure the inference performance of the FP16 and INT8 models, we use [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html), OpenVINO's inference performance measurement tool. Benchmark tool is a command line application that can be run in the notebook with `! benchmark_app` or `%sx benchmark_app`.\n",
```

Comment:
```suggestion
    "To measure inference performance of the FP16 and INT8 models, we use OpenVINO's benchmarking solution, the [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html). It can be run in the notebook with: `! benchmark_app` or `%sx benchmark_app`.\n",
```